mirrord/config/src/fs/mode.rs

```diff
@@ -38,6 +36,7 @@ pub enum FsModeConfig {
     #[default]
```

Comment:
Weirdly inconsistent, I feel like I'm missing something. Shouldn't it be ReadOnly, ReadWrite, Local, and Disabled?

Reply:
I missed the enum here with Disabled (as it wasn't before)
I don't want to change the field name itself though as it will be a breaking change.